operator/cmd/mesh/install_test.go

```diff
@@ -32,3 +32,16 @@ func TestInstallEmptyRevision(t *testing.T) {
 	err := rootCmd.Execute()
 	g.Expect(err).To(gomega.HaveOccurred())
 }
+
+func TestInstallInvalidRevision(t *testing.T) {
```

Comment:
nit: could you also add a test using a valid revision?

Reply:
Done.